src/main/reactjs/src/interfaces/authorisation.ts

```diff
@@ -3,6 +3,7 @@ import { WithId } from 'interfaces/withId';
 import { WithVersion } from 'interfaces/withVersion';
 
 export type AuthorisationBasis = 'AUT' | 'KKT' | 'VIR';
+export type PermissionToPublishBasis = 'Kyllä' | 'Ei';
```

Comment:
Näitä voisi muuttaa enumiksi ja käyttää sillä tavalla koodissa.

Reply:
Tehty